### PR TITLE
Fix: meeting important dates can have markdown links

### PR DESCRIPTION
**Pull request by evyncke**
To address feat #3911 

Added a filter for the Django templates: 'markdown'

Data migration includes changing the important dates description to include the right markdown links. 

Still to do (another issue to be created?) as some links in important dates include the actual meeting number :-(

### CODE REVIEW

File: ietf/doc/templatetags/ietf_filters.py [hunk 1]
```diff
@@ -228,6 +229,26 @@ def link_other_doc_match(match):
...
+        python_markdown.markdown(
+            string,
+            extensions=[
+                "extra",
+                "nl2br",
+                "sane_lists",
+                "toc",
+            ],
+        )
+    )
```
Comment (larseggert) [inline ietf/doc/templatetags/ietf_filters.py:247]: Why don't you call the code in ietf/utils/markdown.py?

Comment (evyncke) [reply ietf/doc/templatetags/ietf_filters.py:247]: Because of a loop between the import statement (and no clue how to break this loop)

Comment (evyncke) [reply ietf/doc/templatetags/ietf_filters.py:247]: In the same vein, I am unsure whether "toc" & "nl2br" extensions are useful or dangerous

**Comment by rjsparks** [comment]
I pushed a commit that moves the new filter to avoid the circular import (and simplifies it to use utils.markdown).

Tests don't pass, but it's not this PRs fault.

File: ietf/doc/templatetags/ietf_filters.py (hunk 1 above)
Comment (larseggert) [reply ietf/doc/templatetags/ietf_filters.py]: https://stackoverflow.com/a/7199514/2240756

**Comment by evyncke** [comment]
> I pushed a commit that moves the new filter to avoid the circular import (and simplifies it to use utils.markdown).

thanks, nicer indeed

**Comment by rjsparks** [comment]
See the failed test run - the changes are producing html that doesn't validate.

For example
```
                                 <td>
                                    Cut-off date for revised BOF proposal requests to Area Directors at UTC 23:59. To request a BOF, please see instructions at <a href="https://www.ietf.org/how/bofs/bof-procedures"></a><a href="https://www.ietf.org/how/bofs/bof-procedures&lt;/a>">https://www.ietf.org/how/bofs/bof-procedures</a> on Requesting a BOF.
                                                                                                                                                                 ^
                                                                                                                                                                 wcag/h30: Anchor link must have a text describing its purpose on line 721:162
```

**Comment by jennifer-richards** [comment]
> See the failed test run - the changes are producing html that doesn't validate.
> 

This was happening because the `linkify` filter was confusing the `markdown` filter. Reversing these avoids the problem and I think even does the right thing.

**Comment by evyncke** [comment]
> 
> This was happening because the `linkify` filter was confusing the `markdown` filter. Reversing these avoids the problem and I think even does the right thing.

Indeed ! Thanks

**Pull request merged**